core/vmpu/src/mpu_armv7m/vmpu_armv7m_mpu.c

```diff
@@ -274,6 +311,11 @@ bool vmpu_region_get_for_box(uint8_t box_id, const MpuRegion * * const region, u
     return false;
 }
 
+static bool value_in_range(size_t start, size_t end, size_t value)
```

Comment:
Consider moving `value_in_range` to a header file shared by the vMPU implementations, so they don't each need a copy.